src/reactviews/pages/ConnectionDialog/connectionFormPage.tsx

```diff
@@ -63,9 +62,6 @@ export const ConnectionFormPage = () => {
                     {locConstants.connectionDialog.advancedSettings}
                 </Button>
                 <div className={formStyles.formNavTrayRight}>
-                    <TestConnectionButton
```

Comment:
Is there a reason why we are removing this?

Reply:
It's not wired up to anything yet, so I'm removing it now until I have a chance to actually implement it.